M2/Macaulay2/editors/Makefile.in

```diff
@@ -0,0 +1,26 @@
+# @configure_input@
+include ../../include/config.Makefile
+VPATH = @srcdir@
+ARGS = --script
+
+MADEFILES = M2-symbols.el M2-emacs-help.txt M2-emacs.m2
+SRCFILES = M2-init.el M2-mode.el M2.el README.md
+
+all: @pre_emacsdir@ $(addprefix @pre_emacsdir@/, $(SRCFILES) $(MADEFILES))
+verify: $(addprefix @pre_emacsdir@/, $(SRCFILES) $(MADEFILES)) $(addprefix emacs/, $(SRCFILES) $(MADEFILES)); ls -lrt $^
+Makefile: Makefile.in; cd ../..; ./config.status Macaulay2/editors/Makefile
+
+@pre_emacsdir@ :; @INSTALL@ -d "$@"
+@pre_emacsdir@/% : $(addprefix emacs/, %) ; @INSTALL_DATA@ $^ @pre_emacsdir@
+
+emacs/M2-symbols.el : make-M2-symbols.m2 ../m2/exports.m2 @pre_exec_prefix@/bin/M2@EXE@ @pre_exec_prefix@/bin/M2
+	@pre_exec_prefix@/bin/M2 $(ARGS) @srcdir@/make-M2-symbols.m2 -e 'exit 0'
+emacs/M2-emacs.m2 emacs/M2-emacs-help.txt: emacs/make-M2-emacs-help.m2 @pre_exec_prefix@/bin/M2@EXE@ @pre_exec_prefix@/bin/M2
+	cd emacs; @pre_exec_prefix@/bin/M2 $(ARGS) ../@srcdir@/emacs/make-M2-emacs-help.m2 -e 'exit 0'
```

Comment:
`@srcdir@` is an absolute path, so `../@srcdir@` will not work.  Here's the error message I get at home:
```
make -C $M2BUILDDIR/Macaulay2/editors 
make: Entering directory '/Users/dan/src/M2/M2.git/M2/BUILD/dan/builds.tmp/einsteinium-development/Macaulay2/editors'
cd emacs; /Users/dan/src/M2/M2.git/M2/BUILD/dan/builds.tmp/einsteinium-development/usr-dist/x86_64-Darwin-MacOS-10.14.6/bin/M2 --script ..//Users/dan/src/M2/M2.git/M2/Macaulay2/editors/emacs/make-M2-emacs-help.m2 -e 'exit 0'
../../../../../../../Macaulay2/m2/debugging.m2:20:6:(1):[5]: error: file not found: "/Users/dan/src/M2/M2.git/M2/BUILD/dan/builds.tmp/einsteinium-development/Macaulay2/editors/Users/dan/src/M2/M2.git/M2/Macaulay2/editors/emacs/make-M2-emacs-help.m2"
../../../../../../../Macaulay2/m2/setup.m2:119:11:(1):[4]: --back trace--
../../../../../../../Macaulay2/m2/setup.m2:126:41:(1):[3]: --back trace--
../../../../../../../Macaulay2/m2/setup.m2:129:6:(1):[2]: --back trace--
../../../../../../../Macaulay2/m2/startup.m2.in:547:63:(0):[1]: --back trace--
../../../../../../../Macaulay2/m2/startup.m2.in:668:6:(0): --back trace--
make: *** [Makefile:19: emacs/M2-emacs-help.txt] Error 1
make: Leaving directory '/Users/dan/src/M2/M2.git/M2/BUILD/dan/builds.tmp/einsteinium-development/Macaulay2/editors'
```

Removing the `../` seems to fix it.

Reply:
I'll make the change for you.

Reply:
how did it pass the tests despite this??